src/org-units-selector/OrgUnitsSelector.jsx

```diff
@@ -76,7 +79,7 @@ export default class OrgUnitsSelector extends React.Component {
                       paging: false,
                       fields: "id,displayName",
                   }),
-            getDefaultRoots(props.d2),
+            this.getRoots({}),
```

Comment:
Since the default call to getRoots is with "{}", wouldn't it make sense to change the signature to ```getRoots({ filter } = {})``` and leave here a ```this.getRoots();```.

Reply:
Yes!